cmd/nvidia-ctk-installer/main.go

```diff
@@ -70,13 +75,15 @@ func main() {
 type app struct {
 	logger logger.Interface
 
-	toolkit *toolkit.Installer
+	nriPlugin *nri.Plugin
```

Comment:
Do we really need to embed this here? Does it make sense to just instantiate it when required and use a deferred shutdown?